README.md

```diff
@@ -62,6 +62,21 @@ BeeswaxAPI::App.configure do |config|
 end
 ```
 
+Example of using:
+
+```ruby
+  params = {
+    advertiser_id: ENV["BEESWAX_ADVERTISER_ID"].to_i,
+    creative_asset_name: file_name.html,
+    size_in_bytes: 12001,
+    notes: "Created by API",
+    active: false
+  }
+  BeeswaxClientWrapper.html_asset(body_params: params)
```

Comment:
why are we referring to a beeswax wrapper here? Maybe it can be confusing for future developers looking at the readme (I've also seen it in the code as well)